frontend/src/libs/helpers/get-formatted-time.helper.ts

```diff
@@ -1,22 +1,13 @@
-const SECONDS_IN_MINUTE = 60;
-const TARGET_STRING_LENGTH = 2;
+import { format } from 'date-fns';
 
-const getFormattedTime = (time: number): string => {
-  if (time && !Number.isNaN(time)) {
-    const minutes = Math.floor(time / SECONDS_IN_MINUTE);
-    const formattedMinutes = minutes
-      .toString()
-      .padStart(TARGET_STRING_LENGTH, '0');
-
-    const seconds = Math.floor(time % SECONDS_IN_MINUTE);
-    const formattedSeconds = seconds
-      .toString()
-      .padStart(TARGET_STRING_LENGTH, '0');
+const EPOCH_MILLISECONDS = 0;
+const TIME_FORMAT = 'mm:ss';
 
-    return `${formattedMinutes}:${formattedSeconds}`;
-  }
+const getFormattedTime = (time: number): string => {
```

Comment:
It should receive date and format as inputs and define a set of available format in helper

Reply:
For now, I decided to change this code to this. Because in progressBar I receive time in seconds, and Date expects to get milliseconds. Although it gets time in the local timezone, so it adds +3 hours for me. But I don't need it, because I need to set a time duration which differs from date.  If I pass the date from the progressBar I need to somehow deal with this +3 hours. So the best way I can imagine this late at night - is this =)
So, if you know, how could I improve it, please help me.
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/102380564/da684a0b-304b-4b8c-97dc-cb72f4e279c0)